src/pages/Dashboards/dashboards/EthStakingPools/EthStakingPools.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import cx from 'classnames'
+import Info from '../shared/Info/Info'
+import EthStakingPools from '../../../../ducks/Eth2.0/EthStakingPools/EthStakingPools'
+import DistributionBtcOnEth from './DistributionBtcOnEth/DistributionBtcOnEth'
+import { withRenderQueueProvider } from '../../../../ducks/renderQueue/viewport'
+import dashboardsStyles from '../dashboards.module.scss'
+
+const EthStakingPoolsPage = () => (
+  <section className={cx(dashboardsStyles.wrapper, 'column')}>
+    <Info
+      title='Ethereum Staking Pools'
+      description='Information all about staking metrics and statistics for the new Ethereum 2.0.'
+    />
+    <main className={cx(dashboardsStyles.content, 'column')}>
+      <div id='eth_validators'>
+        <h4 className='h4 txt-b mrg-xxl mrg--b'>Number of Validators</h4>
+        <div>
+          <EthStakingPools />
+        </div>
+      </div>
```

Comment:
Create `Section` component with `id`, `title` and `children` props. Reuse it in all dashboards